src/App.js

```diff
@@ -9,7 +10,11 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <Article
+    title={article.title}
+    text={article.text}
+    date={article.date}
```

Comment:
```suggestion
    date={article.date}
    text={article.text}
```

Reply:
You always have this sequence **title** => **date** => **text**
But not here!
I believe that order should be followed, even if it does not affect the end of result

Reply:
sequence is not important in this case